src/test/app/AmendmentTable_test.cpp

```diff
@@ -158,7 +195,8 @@ class AmendmentTable_test final : public beast::unit_test::suite
         return makeTable(
             env.app(),
             majorityTime,
-            makeDefaultYes(supportedYes_),
+            makeDefaultYes(yes_) + makeDefaultNo(enabled_) +
+                makeDefaultYes(vetoed_) + makeObsolete(obsolete_),
```

Comment:
> I'm puzzled why you're calling `makeDefaultYes(vetoed_)`. I would have expected that to be `makeDefaultNo(vetoed_)`. At the very least I think it deserves a comment?

Yeah, I did that intentionally so that I could be sure that later operations were doing a thing, and not just using the default vote. I agree it's non-intuitive, so I added a comment, as you suggested.

Reply:
> Also, this use of `std::vector::operator+` needs to be replaced as well. The best I came up with is this:

Yeah, that was a quick and dirty thing just to see if it would work. I rewrote it to use an explicit function call with variable parameters that should be a little safer, and more efficient.